aptos-move/framework/aptos-framework/sources/configs/staking_config.move

```diff
@@ -192,6 +338,46 @@ module aptos_framework::staking_config {
         assert!(minimum_stake <= maximum_stake && maximum_stake > 0, error::invalid_argument(EINVALID_STAKE_RANGE));
     }
 
+    fun validate_rewards_config(
+        rewards_rate: u64,
+        min_rewards_rate: u64,
+        rewards_rate_denominator: u64,
+        rewards_rate_period_in_micros: u64,
+        rewards_rate_decrease_rate_bps: u64,
+    ) {
+        assert!(
+            rewards_rate_denominator > 0,
+            error::invalid_argument(EZERO_REWARDS_RATE_DENOMINATOR),
+        );
+
+        // `rewards_rate` and `min_rewards_rate` which are the numerator are limited to
+        // be `<= MAX_REWARDS_RATE` in order to avoid the arithmetic overflow in the rewards calculation.
+        // `rewards_rate_denominator` can be adjusted to get the desired rewards rate
+        // (i.e., rewards_rate / rewards_rate_denominator).
+        assert!(rewards_rate <= MAX_REWARDS_RATE, error::invalid_argument(EINVALID_REWARDS_RATE));
+        assert!(min_rewards_rate <= MAX_REWARDS_RATE, error::invalid_argument(EINVALID_REWARDS_RATE));
+
+        assert!(
+            min_rewards_rate <= rewards_rate,
```

Comment:
Why not strictly smaller? Equal doesn't seem like a valid case

Reply:
Can you elaborate why we prefer strictly smaller? Maybe I miss something. In [token economic](https://aptosfoundation.org/currents/aptos-tokenomics-overview), it uses a _lower bound of 3.25% annually_. My understand is that lower bound means <=.

Reply:
That's true. I was thinking if reward rate = min reward rate, the other configs (with potentially new values) would not mean anything as there'd be no reduction